src/mad_spark_alt/evolution/fitness.py

```
@@ -1,23 +1,16 @@
 """
 Fitness evaluation for genetic evolution.
 
-This module integrates with the existing creativity evaluation infrastructure
-to provide fitness scoring for ideas in the genetic algorithm.
+This module uses the unified evaluation system to provide consistent
+fitness scoring for ideas using the 5-criteria system.
 """
 
 import asyncio
 import logging
 from typing import Dict, List, Optional
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)